core/rio/trig/src/main/java/org/eclipse/rdf4j/rio/trig/TriGWriter.java

```diff
@@ -164,7 +166,7 @@ protected void closeActiveContext() throws IOException {
 		}
 	}
 
-	private static final boolean contextsEquals(Resource context1, Resource context2) {
+	private static boolean contextsEquals(Resource context1, Resource context2) {
```

Comment:
I have no objection (final is pretty much pointless on a private method) but wonder why you did this? Is this a code cleanup thing from your IDE kicking in?

Reply:
All good, it's trivial enough. I was just wondering :)